imports/plugins/core/orders/client/components/invoice.js

```diff
@@ -0,0 +1,154 @@
+import React, { Component, PropTypes } from "react";
+import { formatPriceString } from "/client/api";
+import { Translation } from "/imports/plugins/core/ui/client/components";
+import DiscountList from "/imports/plugins/core/discounts/client/components/list";
+
+class Invoice extends Component {
+  constructor(props) {
+    super(props);
+
+    this.renderConditionalDisplay = this.renderConditionalDisplay.bind(this);
```

Comment:
Do these need to be bind-ed to the component instance? You should only need to do that in the case of event handlers for `onClick` and etc.

And in those cases you can do the following as we have class properties enabled through babel.

```
myFunctionName = () => {

}
```